src/test_application/DebugObject.cpp

```diff
@@ -176,14 +180,15 @@ void DebugCameraController::update_physics_post()
             std::cout << "next\n";
         }
 
-        targetValid = m_scene.get_registry().valid(m_orbiting);
+        targetValid = rReg.valid(m_orbiting);
 
         if (targetValid)
         {
             // enable the first MachineUserControl
             ActiveEnt firstPart = *(view.get(m_orbiting).m_parts.begin());
 
-            m_scene.reg_get<MachineUserControl>(firstPart).enable();
+            auto* pMUserCtrl = rReg.try_get<MachineUserControl>(firstPart);
+            if (pMUserCtrl != nullptr) { pMUserCtrl->enable(); }
```

Comment:
if(auto* pMUserCtrl = rReg.try_get<MachineUserControl>(firstPart);
   pMUserCtrl != nullptr)
 { pMUserCtrl->enable(); }
 
 I guess it could go either direction. I imagine ya'll won't like this version.

Reply:
I guess it's just what you're used to, but most of the time I find multi-statement `if` conditions equally or less readable than their normal counterparts. Particularly when squeezing them together causes the condition to run over multiple lines, it just becomes more difficult to tell which parts are the condition and which are the body. I'll do it in some places so continue suggesting it, but I'm going to leave it as-is in this particular instance.